.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,38 @@
+name: Publish pydoctor to PyPI
+on:
+  push:
+    tags: 
+      - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
```

Comment:
can we pin a specific checkout version? ... maybe in the future master will introduce backward incompatible changes